addons/mail/static/src/core/web/discuss_sidebar_mailboxes.xml

```diff
@@ -16,12 +16,10 @@
     </t>
 
     <t t-name="mail.Mailbox">
-        <button
-            class="o-mail-DiscussSidebar-item btn d-flex align-items-center py-1 px-0 mx-2 border-0 rounded-0 fw-normal text-reset"
-            t-att-class="{
-                'bg-inherit': mailbox.notEq(store.discuss.thread),
-                'o-active': mailbox.eq(store.discuss.thread),
-            }"
+        <a
+            role="button"
```

Comment:
Not really a button in this case, this is an actual link changing page (especially if fixing href).

Reply:
That is actually a good idea, I look into that.